src/functions/users/flextime/handler.ts

```diff
@@ -32,34 +22,57 @@ export const listUsersFlextimeHandler: APIGatewayProxyHandler = async (event) =>
     const usersWithFlextime = await Promise.allSettled(
       optedInUsers.map(async (severaUser) => {
         try {
+          /**
+           * TODO: This should be typed according to the spec response type for the lambda function
+           * Spec in the lambdas for the futur
+           */
           const flextime = await api.getFlextimeBySeveraUserId(severaUser.guid);
           return {
             user: {
               id: severaUser.guid,
               firstName: severaUser.firstName || "",
               lastName: severaUser.lastName || "",
-              email: severaUser.email || "",
-              attributes: {
-                severaUserId: severaUser.guid,
-                isActive: true
-              }
             },
             flextime: {
               totalFlextimeBalance: flextime?.totalFlextimeBalance || 0,
               monthFlextimeBalance: flextime?.monthFlextimeBalance || 0
             }
           };
         } catch (error) {
-          throw error;
+          /**
+           * Per-user error handling
+           */
+          console.error(`Error fetching flextime for user ${severaUser.guid}:`, error);
```

Comment:
no need for the console.error here as we are returning the message later in the catch.
Also we don't need TSDOC comments over a console.error and I'd say we don't actually need any comment here.